xtras/src/address_map.rs

```diff
@@ -63,6 +66,21 @@ where
         Ok(())
     }
 
+    pub async fn send_async_safe<M>(&self, key: &K, msg: M) -> Result<(), NotConnected>
```

Comment:
I think we can drop the `_safe` here. That only exists on the trait to differentiate it from `xtra`'s `send_async` function.